### DISCUSSION TITLE
Hi, is there any solution for unity apple signin in android?

### DISCUSSION THREAD

**Discussion by heyChenfu** [lupidan/apple-signin-unity#124] [General]
I saw apple-signin-unity only support ios, tvos, osx. But my unity project wants apple signin both at ios and android, I don't know how to do this.

**Comment by lupidan** [comment]
This plugin is for supporting the native iOS SDK to sign in with apple.
For other platforms, you probably need som OAuth/OpenID solution, or a solution on your own

Some stuff to read:

https://sarunw.com/posts/sign-in-with-apple-4/
https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_rest_api
